### PR TITLE
added support for ip address anonymization

### PR DESCRIPTION
**Pull request by ingooooooo**
I added a new option 

```
:anonymizeIp => true
```

for ip address anonymization. This is useful for sites hosted in Germany in order to meet legal requirements as described under #4 in the following document (in German):

http://www.datenschutz-hamburg.de/uploads/media/GoogleAnalytics_Hinweise_fuer_Webseitenbetreiber_in_Hamburg_01.pdf

see also the api docs: 

https://developers.google.com/analytics/devguides/collection/gajs/methods/gaJSApi_gat?hl=de#_gat._anonymizeIp

### CODE REVIEW

**Comment by tmaier** [comment]
Thank you @ingooooooo. I also need this because of the german data protection laws.
Would be great if @leehambley or @smontgomerie could review and merge it.

**Comment by leehambley** [comment]
I'm fine with that being merged, but I don't have time to take responsibility for another release at the moment.

**Comment by tmaier** [comment]
This is fine @leehambley.Thank you.

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ config.middleware.use Rack::GoogleAnalytics, :tracker => 'UA-xxxxxx-x'
 * `:async`      -  sets to use asynchronous tracker
 * `:multiple`   -  sets track for multiple subdomains. (must also set :domain)
 * `:top_level`  -  sets tracker for multiple top-level domains. (must also set :domain)
+* `:anonymizeIp` -  sets the tracker to remove the last octet from all IP addresses, see https://developers.google.com/analytics/devguides/collection/gajs/methods/gaJSApi_gat?hl=de#_gat._anonymizeIp for details.
```
Comment (tmaier) [inline README.md:42]: A symbol with camel case, like `:anonymizeIp`, is not very common.
I would suggest to change it to underscore, like `:anonymize_ip`.

This is also how they name it in middleman-google-analytics gem.
https://github.com/MrJoy/middleman-google-analytics/blob/master/lib/middleman-google-analytics/extension.rb

Comment (smontgomerie) [inline README.md:42]: Agreed.

**Pull request merged**